SourceX/storm/storm.cpp

```diff
@@ -733,6 +748,9 @@ BOOL SVidPlayContinue(void)
 		SDL_Surface *tmp = SDL_ConvertSurface(SVidSurface, window->format, 0);
 		// NOTE: Consider resolution switching instead if video doesn't play
 		// fast enough.
+#if D_BPP == 8
+		pal_surface_offset = { 0, 0, SVidWidth, SVidHeight };

```

Comment:
Will SDL handel centering even if the device was not able to switch to an exact matching resolution?

Reply:
Yes